### PR TITLE
Add Homebrew installation details

### PR DESCRIPTION
**Pull request by cdrage**
**What type of PR is this:**

/kind documentation

**What does this PR do / why we need it:**

Adds documentation on how to install via `brew`.

**Which issue(s) this PR fixes:**

Fixes https://github.com/redhat-developer/odo/issues/5801

**PR acceptance criteria:**

- [X] Documentation

**How to test changes / Special notes to the reviewer:**

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: docs/website/docs/getting-started/installation.md [hunk 1]
```diff
@@ -224,6 +225,23 @@ export PATH=$PATH:$HOME/bin
 # (Optional) Add the $HOME/bin to your shell initialization file
 echo 'export PATH=$PATH:$HOME/bin' >> ~/.bashrc
 ```
+</TabItem>
+<TabItem value="homebrew">
+
```
Comment (rnapoles-rh) [inline docs/website/docs/getting-started/installation.md:230]: Should we mention that this installs the latest V2.x version?

Comment (cdrage) [reply docs/website/docs/getting-started/installation.md:230]: Nope, as by default like the other installation links, it installs the latest *stable* which should be 2.X

Once we switch to 3.0.0 that'll change however.

**Review by feloy — APPROVED** [review]

**Comment by feloy** [comment]
/override OpenShift Integration tests/OpenShift Integration tests

Doc only

**Pull request merged**